### PR TITLE
lzop: test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/lzop.rb [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+    system "make", "install"
+  end
+
+  test do
+    path = testpath/"test"
+    text = "This is Homebrew"
+    path.write text
+
+    system "#{bin}/lzop", "test"
+    assert File.exist? "test.lzo"
```
Comment (xu-cheng) [inline Library/Formula/lzop.rb]: ` assert File.exist?("test.lzo")`

File: Library/Formula/lzop.rb [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+    rm path
+
+    system "#{bin}/lzop", "-d", "test.lzo"
+    assert File.exist? path
```
Comment (xu-cheng) [inline Library/Formula/lzop.rb]: This is unnecessary. `path.read` in below will check it.

Comment (bfontaine) [inline Library/Formula/lzop.rb]: Thanks, removed.

File: Library/Formula/lzop.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/lzop.rb]: Those are optional here. I added them, but why?

Comment (xu-cheng) [inline Library/Formula/lzop.rb]: I thought it would cause ruby ambiguities warning. So I double checked it. It didn't. My bad.

**Comment by xu-cheng** [comment]
Thanks @bfontaine :ship:ed.

**Pull request closed without merging**